sdk/go/common/workspace/plugins.go

```diff
@@ -897,6 +989,14 @@ type PluginSpec struct {
 	Checksums map[string][]byte
 }
 
+func (p *PluginSpec) PackageDescriptor() string {
```

Comment:
I think we could pick a better name than `PackageDescriptor` here, since this isn't really a `PackageDescriptor` in the sense that the type is defined elsewhere. Something like `FullName`, `LocalName`, `URI`, maybe? 🤔